src/activities/Elsa.Activities.Timers.Hangfire/Extensions/TimeSpanExtensions.cs

```diff
@@ -0,0 +1,19 @@
+namespace System
+{
+    public static class TimeSpanExtensions
+    {
+        public static string ToCronExpression(this TimeSpan timeSpan)
```

Comment:
Is there a specific reason not to do this for `Duration`?

Reply:
Has no reason. It was created during the development. I had it differently in the beginning.